src/parameters/ParamValue.ts

```diff
@@ -7,17 +7,22 @@ class ParamValue {
       this.addValue(paramValue);
     }
   }
+
   toString() {
     return this.values.join(this.delimiter);
   }
 
   addValue(value: unknown) {
+    //Append values or array of values
     if (Array.isArray(value)) {
       this.values = this.values.concat(value);
     } else {
       this.values.push(value);
     }
 
+    //remove empty values
+    this.values = this.values.filter((v) => v != null);
```

Comment:
What do you think of 
this.values.filter((v) => v); (remove false, null, undefined, '' any aother falsy element)

Reply:
agree